packages/@azure/servicebus/data-plane/lib/core/messageReceiver.ts

```diff
@@ -81,10 +81,11 @@ export interface ReceiveOptions extends MessageHandlerOptions {
    */
   receiveMode?: ReceiveMode;
   /**
-   * @property {number} [maxConcurrentCalls] The maximum number of messages that should be
-   * processed concurrently while in peek lock mode. Once this limit has been reached, more
-   * messages will not be received until messages currently being processed have been settled.
-   * - **Default**: `1` (message at a time).
+   * @property {number} [maxConcurrentCalls] The maximum number of messages that can be
+   * fetched over the network concurrently while in peek lock mode.
+   * This setting can be customized to take a higher number if the Service Bus client is
+   * running on a multi-core platform.
```

Comment:
Service Bus client or the Service bus service?

The word `client` would generally mean the sdk and the application the customer is writing.
Where as we want to refer to the machine where Service Bus is running, correct?

On second thoughts, do we even want to mention this?

I believe the previous sentence sums up the feature very nicely. "Fetched over the network".

Same for `maxConcurrentCallsPerSession`

Reply:
The intent is for the client and application customer is writing. The rhea links would be between the client app and service bus endpoints. Idea is that the messages can be loaded onto this link in parallel.

Reply:
Yes, that's right.
My point was whether the parallel loading is dependent on the platform where the user's code is deployed or on the platform where the service is deployed

Reply:
Parallel loading would depend on the platform where the user's code is deployed, as that would be where the rhea credit management code is running.